### PR TITLE
fix(curriculum): missing code block 

### LINKED ISSUE #47074: Missing code block on step 31 of building a registration form
### Describe the Issue

in step 31 of building a registration form, the description in line 10 is missing code blocks

>  Nest the select element (with its option elements)

This should be changed to

>  Nest the select element (with its `option` elements)

https://github.com/freeCodeCamp/freeCodeCamp/blob/972dbd5c27579ffc77c10acc6fec8170943cf1dd/curriculum/challenges/english/14-responsive-web-design-22/learn-html-forms-by-building-a-registration-form/60fac56271087806def55b33.md?plain=1#L10

### Additional context

If you would like to fix this issue, please make sure you read [our guidelines for contributing](https://contribute.freecodecamp.org/#/), we prioritize contributors following the instructions in our guides. Join us in [our chat room](https://chat.freecodecamp.org/channel/contributors) or [the forum](https://forum.freecodecamp.org/c/contributors/3) if you need help contributing, our moderators will guide you through this.

Sometimes we may get more than one pull requests. We typically accept the most quality contribution, followed by the one that is made first.

Happy contributing.

### PR DESCRIPTION
**Pull request by vedanshiprajapati**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #47074 

this pull request does the changes that is required for the issue #47074

### CODE REVIEW

**Comment by Sboonny** [comment]
@Vedanshi555 please follow the guideline https://contribute.freecodecamp.org/#/index

**Comment by Vedanshi555** [comment]
> @Vedanshi555 please follow the guideline https://contribute.freecodecamp.org/#/index
> 
> I have added the boilerplate back

sorry, i didn't get it 
nd i think i did follow the guideline:)

**Comment by Vedanshi555** [comment]
closes #47074

@Sboonny

**Review by Sboonny — DISMISSED** [review]
#### :sparkles: :tada: **AWESOME!** :tada: :sparkles:

 Hi @Vedanshi555, thanks for this pull request and for contributing to the code-base for the first time. We are looking forward to more contributions from you in the future.

 Cheers & happy contributing!

File: curriculum/challenges/english/14-responsive-web-design-22/learn-html-forms-by-building-a-registration-form/60fac56271087806def55b33.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ dashedName: step-31
 
 # --description--
 
-Nest the select element (with its option elements) within a `label` element with the text `How did you hear about us?`. The text should come before the `select` element.
+Nest the select element (with its `option` elements) within a `label` element with the text `How did you hear about us?`. The text should come before the `select` element.
```
Comment (jeremylt) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-html-forms-by-building-a-registration-form/60fac56271087806def55b33.md:10]: This one needs code formatting too, right?

```suggestion
Nest the `select` element (with its `option` elements) within a `label` element with the text `How did you hear about us?`. The text should come before the `select` element.
```

**Review by Sboonny — APPROVED** [review]

**Comment by Vedanshi555** [comment]
@jeremylt 
can you approve this pr?

**Review by jeremylt — APPROVED** [review]

**Pull request merged**